spec/firebase_app_distribution_remove_testers_action_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'fastlane/action'
+require 'fastlane_core/ui/ui'
+
+describe Fastlane::Actions::FirebaseAppDistributionRemoveTestersAction do
+  let(:action) { Fastlane::Actions::FirebaseAppDistributionRemoveTestersAction }
+  describe '#run' do
+    before(:each) do
+      allow(action).to receive(:fetch_auth_token).and_return('fake-auth-token')
+    end
+
+    it 'raises an error if emails and file are blank' do
+      expect { action.run({}) }
+        .to raise_error("Must specify `emails` or `file`.")
+    end
+
+    it 'raises an error if there are > 1000 emails' do
+      emails = (1..1001).map { |i| "#{i}@e.mail" }.join(',')
+
+      expect { action.run({ emails: emails }) }
+        .to raise_error("A maximum of 1000 testers can be removed at a time.")
+    end
+
+    it 'succeeds and makes call with value from emails param' do
+      project = 1
+      emails = '1@e.mail,2@e.mail'
+      path = 'path/to/file'
+      count = 1
+      fake_file = double('file')
+      allow(File).to receive(:open)
+        .with(path)
+        .and_return(fake_file)
+      allow(fake_file).to receive(:read).and_return('')
```

Comment:
Instead of allowing this read, can we expect the file not to receive `:read` in this case? Or just not mock out any of this file related stuff.

Reply:
Good call! Done.